defaults/main.yml

```diff
@@ -3,14 +3,14 @@
 oc_mirror: https://github.com/openshift/origin/releases/download
 oc_platform: linux-64bit
 
-oc_ver: v3.6.1
-oc_sha: '008f2d5'
+oc_ver: v3.7.0
+oc_sha: '7ed6862'
 
 oc_checksums:
-  v3.6.1:
-    '008f2d5':
-      linux-64bit: sha256:922afb7a5642040ea7a6b780cd68eb1d15533b6376b503351a4c38a452338d11
-      mac: sha256:b7ecd27a0e3821868255898ef0ab47dfe0e5e371da6a58b0cbadd687e8dccca0
+  v3.7.0:
+    '7ed6862':
+      linux-64bit: sha256:573e39965633671aac08d68f57d66ff182fb646649523914f0a60debb6515eba
+      mac: sha256:7b00cf3535e9b5a33c30a143bdb5621fe49f73aee63c77ff4a7f8d0212be4f0a
 
```

Comment:
please leave the old version in there. the role can support a default version while containing checksums for multiple versions.